stream_feed_dart/lib/src/client/feed/flat_feed.dart

```diff
@@ -45,6 +45,7 @@ class FlatFeet extends Feed {
     final options = {
       'limit': limit ?? Default.limit,
       'offset': offset ?? Default.offset,
+      ...flags?.params ?? Default.enrichmentFlags.params,
```

Comment:
Hey, nice catch. Can you just move this after `filter params`

Reply:
Have pushed the fix.

Reply:
Thanks 👍